doc/launch.md

```diff
@@ -0,0 +1,119 @@
+# Server startup
+
+When an EAP server provisioned with the cloud feature-pack is started, a set of bash scripts are executed in order to adjust 
+the server configuration. These bash scripts are controlled by a set of environment variables that this document describes.
+
+When running an EAP server inside the EAP S2i runtime and builder images, you can use this [these environment variables](https://github.com/jboss-container-images/openjdk/blob/develop/modules/jvm/api/module.yaml) to configure the Java VM.
+
+EAP S2i runtime and builder images are exposing a set of [environment variables](https://github.com/wildfly/wildfly-cekit-modules/blob/main/jboss/container/wildfly/run/api/module.yaml) to fine tune the server execution.
+
+# Datasources
+
+Configure datasources subsystem.
+
+## Environment variables
+
+[Environment variables.](https://github.com/wildfly/wildfly-cekit-modules/blob/main/jboss/container/wildfly/launch/datasources/module.yaml)
```

Comment:
Note related to your PR but we need to create an index of all these environment variables...